### PR TITLE
Final Edits on Purchase Request v4

### PR DESCRIPTION
**Pull request by sarraloew**

### CODE REVIEW

**Comment by sarraloew** [comment]
:shipit:

File: src/api-reference/invoice/v4.purchase-request-endpoints.markdown [hunk 1]
```diff
@@ -67,7 +79,7 @@ ToDo: Pull from documentation template.
 
 #### Request
 
-This is just a sample set of fields. The fields and values needing to be passed for your entity will vary based on your edition of SAP concur and your forms and fields setup, but should include most of these common fields.
+This is a sample set of fields. The fields and values  your entity requires will vary based on your edition of Concur Invoice, and your forms and fields configuration. This example includes commonly used fields.
```
Comment (charlieconcur) [inline src/api-reference/invoice/v4.purchase-request-endpoints.markdown]: You've got two spaces in between 'values' and 'your. I've used the suggestion feature -- you should just be able to click a button to accept the fix.

```suggestion
This is a sample set of fields. The fields and values your entity requires will vary based on your edition of Concur Invoice, and your forms and fields configuration. This example includes commonly used fields.
```

**Review by charlieconcur — APPROVED** [review]
Looks good. One super-nitpicky thing and we are good to go.

**Comment by sarraloew** [comment]
All right, let's publish this one! :shipit:

**Pull request merged**